### ISSUE TITLE
There is not a good way to call trtllm backend to initialize lora weights from Python BLS

### ISSUE THREAD

**Issue by ShuaiShao93** [triton-inference-server/server#7856]
**Is your feature request related to a problem? Please describe.**
We want to call trtllm backend from a BLS model to initialize the lora weights before it starts serving the user request. I tried to do that in `initialize()`, but it failed because it can't call another model before server is ready.

**Describe the solution you'd like**
Add another function `initialize_after_server_is_ready()`

**Describe alternatives you've considered**
Initialize lora weights when we see the first user request, but this harms the latency

**Comment by rmccorm4** [comment]
Hi @ShuaiShao93,

I believe the Python Backend model load APIs were added for this exact purpose so the BLS model can check or load its dependent models directly before using them: https://github.com/triton-inference-server/python_backend?tab=readme-ov-file#model-loading-api. Can you try this?

**Comment by ShuaiShao93** [comment]
> Hi [@ShuaiShao93](https://github.com/ShuaiShao93),
> 
> I believe the Python Backend model load APIs were added for this exact purpose so the BLS model can check or load its dependent models directly before using them: https://github.com/triton-inference-server/python_backend?tab=readme-ov-file#model-loading-api. Can you try this?

Yeah that was exactly what I used. From the link you can see `the model loading API should only be used after the server has been running`, so it doesn't work as I want.

<img width="867" alt="Image" src="https://github.com/user-attachments/assets/adc49a29-aaff-4e92-8e70-569aeb61b3b6" />
